resources/datadog-integrations/software.rb.erb

```diff
@@ -8,6 +8,7 @@ if windows?
   require_relative "../../resources/datadog-integrations/validate_manifest"
 else
   require_relative "<%= project_dir %>/resources/datadog-integrations/validate_manifest"
+  whitelist_file /\/opt\/datadog-agent\/.+\.so.+/
```

Comment:
I don't think there are any anywhere else, but covering everything wouldn't be harmful, would it?